File: Lab2KarpaBohdan36/src/Lab2KarpaBohdan36/Root.java
package Lab2KarpaBohdan36;

/**
 * Клас, що представляє корінь рослини.
 */
public class Root {
    private String rootType;

    /**
     * Конструктор, що ініціалізує тип кореня.
     * @param rootType Тип кореня (мочкуватий або стрижневий).
     */
    public Root(String rootType) {
        this.rootType = rootType;
    }

    /**
     * Метод для отримання типу кореня.
     * @return Тип кореня.
     */
    public String getRootType() {
        return rootType;
    }

    /**
     * Метод для зміни типу кореня.
     * @param rootType Новий тип кореня.
     */
    public void setRootType(String rootType) {
        this.rootType = rootType;
    }

    @Override
    public String toString() {
        return "Тип кореня: " + rootType;
    }
}

File: plants/root.py
class Root:
    """Клас, що представляє корінь рослини."""

    def __init__(self, root_type):
        """Ініціалізує тип кореня."""
        self.root_type = root_type

    def get_root_type(self):
        """Повертає тип кореня."""
        return self.root_type

    def set_root_type(self, root_type):
        """Задає новий тип кореня."""
        self.root_type = root_type

    def __str__(self):
        return f"Тип кореня: {self.root_type}"
